src/bench.c

```diff
@@ -472,7 +506,7 @@ char *benchmark_format(struct fmt_main *format, int salts,
 		wait_salts = 1;
 		benchmark_time *= -1;
 	}
-	wait = format->params.benchmark_length ? 0 : wait_salts;
+	wait = format->params.benchmark_length & 0xe00 ? wait_salts : 0;
```

Comment:
What's this `wait` thing? Where do we set those bits?

Reply:
That's for `--test=-1` which will benchmark as long as it takes to get a complete many-salts figure.

Reply:
I'll add the braces, thanks